README.md

```diff
@@ -105,34 +95,29 @@ Import the baseclass and define a schema.
         def deserialize_media_type(self, obj):
             return SimpleSchema.MEDIA_TYPE
 
-Define a function that handles messages for the schema:
+Define a function that handles messages for the schema and decorate it with `@handles` to
+indicate that it handles your schema type. While plain functions, suffice, most real-world
+handlers will be a class with its own `@binding` to pass other collaborators:
 
-    def handle_simple(message):
-        print message
-        return True
+    @binding("simple_handler")
+    @handles(SimpleSchema)
+    class SimpleHandler(object):
+        def __init__(self, graph):
+            self.collaborator = graph.collaborator
```

Comment:
Can you please explain what's the purpose of the `collaborator`.
And also, in which layer is is being initialized?